tests/unit/__snapshots__/storybook.spec.ts.snap

```diff
@@ -14,7 +14,156 @@ exports[`Storyshots ConsoleIo/Print Block Existing Data Block 1`] = `
+                              />
+                               
+                              <small
+                                class="text-muted"
+                              >
+                                
+      Only accepts word characters
+    
+                              </small>
+                            </div>
+                          </div>
+                           
+                          <div>
+                            <div
+                              class="form-group"
+                            >
+                              <label>
+                                Block label
+                              </label>
+                               
+                              <textarea
+                                class="form-control"
+                                placeholder="Enter a block label"
+                              />
+                               
+                            </div>
+                          </div>
+                           
+                          <div>
+                            <div
+                              class="form-group"
+                            >
+                              <label>
+                                Block semantic label
+                              </label>
+                               
+                              <textarea
+                                class="form-control"
+                                placeholder="Enter a block semantic label"
+                              />
+                               
+                            </div>
+                          </div>
+                           
+                          <!---->
+                           
+                          <div
+                            class="starting-block-button"
+                          >
+                            <h4>
+                              Options
+                            </h4>
+                             
+                            <div
+                              class="form-group"
+                            >
+                              <button
+                                class="btn btn-secondary btn-sm"
+                                disabled="disabled"
+                                type="button"
+                              >
+                                
+          Currently set as starting block
+        
+                              </button>
+                            </div>
+                          </div>
+                        </fieldset>
+                         
+                        <div
+                          class="text-right"
+                        >
+                          <small
+                            class="text-muted"
+                          >
+                            Block Id: 
+                            <em>
+                              6f67fcec-c5ba-4451-ae76-51a29b6b30fd
```

Comment:
@jcbashdown & @bzabos , it seems like the storyshots is back (before we had empty `<div>`). I don't know what resolved it. But we're having a pbm, for instance, in this particular line, the Uuid in snapshot doesn't match with Uuid during `yarn test:unit`, which will lead to snapshot error.
See current build example: https://travis-ci.com/github/FLOIP/flow-builder/builds/221253135#L352

Any idea how should we handle this ?

Reply:
The uuid generator gives us something new every run. Check this out:

https://github.com/FLOIP/flow-builder/pull/35/files#diff-48aa2e4c8dec9b1a411f6d75bd7208f6e7ff39633b413f167b397aac9319b477R52

But that approach won't work here or will be time consuming to do. We'll need to fake the uuid generator! We'll have to refactor so this is injected or overidden. Should be doable, could you look into it?

I notice there are also some other errors in the build - both in travis and locally for me I see things like:

`"TypeError: Cannot read property 'reduce' of undefined"`

I suspect this is something I saw in my branch which I resolved in the cases I found it by having lots of v-if conditions in my stories:

https://github.com/FLOIP/flow-builder/pull/35/files#diff-e3f6b63837f0b7bef6c0bfd5de701761ba00d57af7911ed3e58b87735e0772bcR27

Reply:
p.s. Really good news the sidebar storyshots are back though!

Reply:
thank for the idea, I will check how we can mock the uuid during tests